shapeshift/web
# feat: fox page plugin routing (do not merge before 1782)

## Description

This implements basic routing for the fox page.
Dependent on https://github.com/shapeshift/web/pull/1782 being merged

## Notice




## Pull Request Type


## Issue (if applicable)

closes https://github.com/shapeshift/web/issues/1763


## Risk



## Testing

- Enable fox page flag
- Browse to fox or fox/fox page => see blank page with "fox tab" text
- Browse to fox/foxy or fox/onefox => see blank page with "foxy tab" text

## Screenshots (if applicable)
